pkg/sbom/sbom.go

```diff
@@ -15,18 +15,35 @@ var (
 	DepGraphWorkflowID = workflow.NewWorkflowIdentifier("depgraph")
 )
 
+const (
+	flagExperimental = "experimental"
+	flagFile         = "file"
+	flagFormat       = "format"
+)
+
 func SBOMWorkflow(
 	ictx workflow.InvocationContext,
 	_ []workflow.Data,
 ) (sbomDocs []workflow.Data, err error) {
 	engine := ictx.GetEngine()
 	config := ictx.GetConfiguration()
+	logger := ictx.GetLogger()
+	format := config.GetString(flagFormat)
+
+	if !config.GetBool(flagExperimental) {
+		return nil, fmt.Errorf("set `--experimental` flag to enable sbom command")
+	}
+
+	logger.Println("sbom workflow start")
```

Comment:
It would be better to be consistent with the style of log messages, e.g. capitalization of SBOM, verb at start vs. at end. Also, is it useful to log both "SBOM workflow start" and "Invoking depgraph workflow"? There's not really anything that can break between those two lines. :)

Reply:
the idea of this message was to shout out that the workflow is commencing. isee now that it is actually misplaced, since it should happen BEFORE the preceeding `if` statement.

the lowercasing was intentinal, since the command is `sbom`. but i agree, it's confusing and seems more like an inconsistency.

Reply:
I see. Personally I think it's OK if the workflow is called the "SBOM workflow" even if the command is written `sbom`. But anyways, these are nits.